packages/pigeon/e2e_tests/test_objc/ios/Runner/MyApi.h

```diff
@@ -4,7 +4,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /// Implementation of the Pigeon generated interface Api.
-@interface MyApi : NSObject<ACApi>
+@interface MyApi : NSObject <ACApi>
```

Comment:
This applies some of the Engine c/objc formatting to the e2e test files. Lmk if you would prefer it to be as is.

Reply:
Yea, this has been an annoyance for me.  The bot seems to differ from the formatter.  Whatever keeps the CI bot happy is good.